source/basic.tex

```diff
@@ -5632,6 +5636,16 @@
 Expressions appearing in the \grammarterm{compound-statement} of a \grammarterm{lambda-expression}
 are not subexpressions of the \grammarterm{lambda-expression}.
 \end{note}
+% FIXME: We never define "potentially-evaluated expression".
```

Comment:
@jensmaurer Does this make you concerned?

Reply:
basic.def.odr p3 defines it: "An expression or conversion is _potentially evaluated_ unless it is ..."

Seems pretty obvious English to go from "an expression is potentially evaluated" to "potentially-evaluated expression". The FIXME should be removed.

Reply:
Sounds good, thanks!